packages/wix-ui-tpa/src/components/Image/Image.st.css

```
@@ -1,13 +1,21 @@
 :import {
   -st-from: "../../common/formatters.st";
-  -st-named: color, fallback;
+  -st-named: color, fallback, font;
```

We can remove `font`:
```suggestion
  -st-named: color, fallback;
```